core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -187,6 +187,11 @@ class DAGScheduler(
   /** If enabled, FetchFailed will not cause stage retry, in order to surface the problem. */
   private val disallowStageRetryForTest = sc.getConf.getBoolean("spark.test.noStageRetry", false)
 
+  /** Number of consecutive fetch failures allowed before a stage is aborted */
+  private[scheduler] val maxConsecutiveFetchFailuresPerStage =
```

Comment:
Ok one more naming nit: can you make this consistent with the config name, so maxConsecutiveStageAttempts (and ditto with the constant)?